### PR TITLE
Cleanup princeton mss PR

### PR DESCRIPTION
**Pull request by aaron-collier**
## Why was this change made?

Replaces #431 

Follow up PRs will be submitted for movie_posters if needed.

@jacobthill this is mostly a copy from your existing PR with a few tweaks and removing the change in the error message that I did (which broke the test). That should get fixed in a separate PR. I would suggest running this against your new princeton data and then merging that into dlme-metadata/master at the same time this is approved/merged. Also note, that I believe the build error here is specifically related to the dlme-metadata repo mismatch, so merging there first, then verifying the build works here before merging is probably the best approach.

## Was the documentation (README, API, wiki, ...) updated?

N/A

### CODE REVIEW

**Comment by jacobthill** [comment]
@aaron-collier I simplified the iiif portion of the princeton config. Please take a look and merge when you get a chance and then I can update the movie posters on a new PR.

File: lib/macros/iiif.rb [hunk 1]
```diff
@@ -87,6 +87,12 @@ def iiif_service_conforms_to(service_profile)
         'http://iiif.io/api/presentation/'
       elsif service_profile.include? 'http://iiif.io/api/search/'
         'http://iiif.io/api/search/'
+      # this doesn't look right, but it's what we get.
+      # can be removed after: https://github.com/pulibrary/figgy/pull/3513
+      elsif service_profile.include? 'http;//iiiif.io/api/image/2/level2.json'
```
Comment (mjgiarlo) [inline lib/macros/iiif.rb]: Looks like the figgy PR has already been merged. Can this be removed now?

File: lib/translation_maps/languages.yaml [hunk 2]
```diff
@@ -182,6 +182,7 @@ thai: Thai
 tigrinya: Tigrinya
 tibetan: Tibetan
 turkish, ottoman: Turkish, Ottoman (1500-1928)
+urkish, ottoman (1500-1928): Turkish, Ottoman (1500-1928)
```
Comment (mjgiarlo) [inline lib/translation_maps/languages.yaml]: Is `urkish` a typo for `turkish`?

File: lib/macros/iiif.rb (hunk 1 above)
Comment (aaron-collier) [reply lib/macros/iiif.rb]: I'll verify, I did not initially see the change applied, so was not sure it was deployed or perhaps cached. But will check.

File: lib/translation_maps/languages.yaml (hunk 2 above)
Comment (aaron-collier) [reply lib/translation_maps/languages.yaml]: Looks like. Will fix.

File: lib/macros/iiif.rb (hunk 1 above)
Comment (aaron-collier) [reply lib/macros/iiif.rb]: Confirmed and removed.

**Pull request merged**